src/js/components/Meter/stories/MultipleValues.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Meter, Stack, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+class MultipleValues extends Component {
+  state = { total: 70 };
```

Comment:
might make more sense to be ` state = { total: 100 }`. unless I'm missing something.

Reply:
Good idea, changed.